util/perl/OpenSSL/Util.pm

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env perl
+# Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+package OpenSSL::Ordinals;
+
+use strict;
+use warnings;
+use Carp;
+
+use Exporter;
+use vars qw($VERSION @ISA @EXPORT @EXPORT_OK %EXPORT_TAGS);
+$VERSION = "0.1";
+@ISA = qw(Exporter);
+@EXPORT = qw(cmp_versions);
+@EXPORT_OK = qw();
+
+=head1 NAME
+
+OpenSSL::Util - small OpenSSL utilities
+
+=head1 SYNOPSIS
+
+  use OpenSSL::Util;
+
+  $versiondiff = cmp_versions('1.0.2k', '3.0.1');
+  # $versiondiff should be -1
+
+  $versiondiff = cmp_versions('1.1.0', '1.0.2a');
+  # $versiondiff should be 1
+
+  $versiondiff = cmp_versions('1.1.1', '1.1.1');
+  # $versiondiff should be 0
+
+=head1 DESCRIPTION
+
+=over
+
+=item B<cmp_versions "VERSION1", "VERSION2">
+
+Compares VERSION1 with VERSION2, paying attention to OpenSSL versioning.
+
+Returns 1 if VERSION1 is greater than VERSION2, 0 if they are equal, and
+-1 if VERSION1 is lesser than VERSION2.
```

Comment:
"less than" not "lesser than"

Reply:
So it's "greater than" but "less than"...

... English, the language where logic said buh-bye :wink: ...

Will fix (approval noted)